### PR TITLE
Add overlayscrollbars w/ npm auto-update via single package.json

### LINKED ISSUE #12562: [Request] Add OverlayScrollbars
**Library name:** OverlayScrollbars
**Library description:** A javascript scrollbar plugin which hides the native scrollbars and provides
custom styleable overlay scrollbars, but keeps the native functionality and feeling.
**Repository url:** https://github.com/KingSora/OverlayScrollbars/
**npm package url:** https://www.npmjs.com/package/overlayscrollbars
**License:** MIT
**Official homepage:** https://kingsora.github.io/OverlayScrollbars/
**Wanna say something? Leave message here:** I would love to have OverlayScrollbars to the cdnjs! I am not the author.

It is a relatively new library which is a high quality one for custom scrollbars on websites with great customization and browser support.

### PR DESCRIPTION
**Pull request by Audiodrome**
close #12562, cc @KingSora

Pull request for issue: #12562 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/KingSora/OverlayScrollbars
 * Official website (optional, not the repository): https://kingsora.github.io/OverlayScrollbars
 * NPM package url (optional): https://www.npmjs.com/package/overlayscrollbars
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 10
   - Count of stars: 118
   - Count of forks: 10
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week: 1,345
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Audiodrome** [comment]
@extend1994 Hi, I could use some help.

So the CI failed and I'm still not sure where the error is coming from. I thought the problem might be my npmFileMap was incorrect. I tried a couple of variations of the node globs but I got the same npm err, when running `npm test` locally. These are the globs I tried.

`"**/*"`
`"**/+(css|js)/*"`
`"+(css|js)"`
`"+(css/js)/*"`

I left the basePath as an empty string since the npm package didn't include a "dist" folder.

**Comment by Audiodrome** [comment]
@PeterDaveHello I found the problem. I had a space between the JSON property name and the colon.

I had this
`"property" : "something"`
instead of
`"property": "something"`

Just ran `npm test`, no errors now. 

Thank you.

File: ajax/libs/overlayscrollbars/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/jquery.overlayScrollbars.js",
+        "js/jquery.overlayScrollbars.min.js",
+        "js/OverlayScrollbars.js",
+        "js/OverlayScrollbars.min.js",
+        "css/OverlayScrollbars.css",
+        "css/OverlayScrollbars.min.css"
```
Comment (extend1994) [inline ajax/libs/overlayscrollbars/package.json]: Let's use [glob](https://github.com/isaacs/node-glob) pattern to fetch all above files:
```js
"+(css|js)/+(*o|O)verlayScrollbars*"
```

File: ajax/libs/overlayscrollbars/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overlayscrollbars",
+  "filename": "OverlayScrollbars.min.js",
```
Comment (extend1994) [inline ajax/libs/overlayscrollbars/package.json]: Should be `js/OverlayScrollbars.min.js` the path to `filename` is the relative path to `basePath`

**Comment by extend1994** [comment]
Thanks @Audiodrom!

**Comment by KingSora** [comment]
@Audiodrome Also a **very** big thanks from me!

**Comment by KoalaBear84** [comment]
When can it be merged? 😇😝 Thanks!

**Comment by PeterDaveHello** [comment]
Going to rebase the branch with the latest master branch and merge it now!

**Pull request merged**